app/services/processing_service.py

```
@@ -712,9 +712,9 @@ async def _get_authenticated_git_client(
         git_config = await self.git_label_repository.find_by_user_and_hosting(
             user_id, git_token, git_provider
         )
-        # TODO : check if needed
-        # if not git_config:
-        #     raise Exception(f"No {git_provider} configuration found for user")
+        
+        if not git_config:
+           raise Exception(f"No {git_provider} configuration found for user")
```

Ruff (0.14.2)

Create your own exception

(TRY002)